src/execution/vm/vm.cpp

```diff
@@ -2060,6 +2060,27 @@ void VM::Interpret(const uint8_t *ip, Frame *frame) {  // NOLINT
   // -------------------------------------------------------
   // String functions
   // -------------------------------------------------------
+  OP(Chr) : {
+    auto *result = frame->LocalAt<sql::StringVal *>(READ_LOCAL_ID());
+    auto *exec_ctx = frame->LocalAt<exec::ExecutionContext *>(READ_LOCAL_ID());
+    auto *input = frame->LocalAt<const sql::Integer *>(READ_LOCAL_ID());
+    OpChr(result, exec_ctx, input);
+    DISPATCH_NEXT();
+  }
```

Comment:
For consistency with the rest of the file, space out the blocks.

Reply:
Fixed